src/main/java/de/tum/in/www1/artemis/service/FileService.java

```diff
@@ -625,7 +628,7 @@ public void convertToUTF8Directory(String startPath) throws IOException {
      * This converts a specific file to the UTF-8 encoding.
      * To determine the encoding of the file, the library com.ibm.icu.text is used.
      *
-     * @param filePath           the path where the file is located
+     * @param filePath the path where the file is located
```

Comment:
Sometimes you erase the space and sometimes you add the space between input and description. Is there a reason for that? Here the space is erased

Reply:
It's my formatter :see_no_evil:

Reply:
:D Alright